js/ui/color_box/color_box.js

```diff
@@ -99,6 +99,12 @@ var ColorBox = DropDownEditor.inherit({
             * @action
             */
 
+            /**
+             * @name dxColorBoxOptions.showDropDownButton
+             * @inheritdoc
+             * @hidden
```

Comment:
DropDown Button for ColorBox may be useful

Reply:
ColorBox has dropDownButton but it cannot be opened by click on the editor because it always accepts text input. If the customer will hide the dropDownButton, they can not open the ColorBox using a mouse